examples/example_mpi.m

```diff
@@ -0,0 +1,13 @@
+function example_mpi()
+% Run from command prompt with: mpiexec -n 2 matlab -batch "cd ABSOLUTE_PATH_TO_MATLABNEURON; setup; example_mpi"
+% On windows, make sure to use neuron's provided mpiexec (usually at C:\nrn\bin\mpiexec.exe).
```

Comment:
I tried the inprocess on linux, but it did not help with not getting a warning that starts with "mpiexec has exited due to process rank 0 with PID 0 on node NODENAME exiting improperly" when using exit(0) as last line here.

And, I do get segmentation violations when running example_mpi and replacing the exit(0) with the neuron.quit (temporarily disabling the override in the matlab code), even if i add the -nojvm option with the -batch option.

So, I think we keep this example as is, and also keep linux outofprocess for now. Note that, since this example is run in batch mode, it is not even needed to do the exit(0) at the end, the batch job will end when all the code has run. However, it is nice to have the explicit 0 return code, to implicitly teach that you could also do non-zero return code. So, update the comment to `% Close this matlab session, returning exit code 0 to signal successful completion`

Reply:
Oh, and add  to the top comment here that on linux / mac you will get that mpiexec warning.

Reply:
Did you try inprocess on linux, while using Neuron's internal `n.quit()`?

Reply:
Yes